lib/util/mailables.js

```diff
@@ -0,0 +1,170 @@
+import blobStream from 'blob-stream'
+import moment from 'moment'
+
+import PDFDocument from './PDFDocumentWithTables'
+
+export const LETTER_FIELDS = [
```

Comment:
Please add some JSDoc noting that the array structure affects the display of these items as elements within rows.